checkout-ui-custom/checkout6-custom.js

```diff
@@ -106,6 +123,29 @@
     })
   }
 
+  const removeCartItemsAndQuote = function () {
+    const orderFormID = window.vtexjs.checkout.orderFormId
+
+    $.ajax({
+      url: `${window.location.origin}/api/checkout/pub/orderForm/${orderFormID}/items/removeAll`,
+      type: 'POST',
+      data: {
+        expectedOrderFormSections: ['items'],
+      },
+    }).then(function () {
+      $.ajax({
+        url: `${window.location.origin}/api/checkout/pub/orderForm/${orderFormID}/customData/b2b-quotes-graphql/quoteId`,
+        type: 'PUT',
+        contentType: 'application/json; charset=utf-8',
+        data: JSON.stringify({
+          value: 0,
+        }),
```

Comment:
Instead of making ajax requests, we could use the `vtexjs` SDK here. I believe `vtexjs.checkout.removeAllItems()` and `vtexjs.checkout.setCustomData()` are the relevant functions.

If there's a reason to stick with ajax, we should add the `rootPath`, similar to what we're doing in the `fetchSettings` function.